packages/venia-concept/.graphqlconfig

```diff
@@ -14,7 +14,8 @@
                     "filesGlob": "../{venia-ui,venia-concept}/{lib,src}/**/*.{js,graphql,gql}",
                     "ignore": [
                         "../venia-ui/lib/**/giftOptions.js",
-                        "../venia-ui/lib/**/wishlistPage.gql.js"
+                        "../venia-ui/lib/**/wishlistPage.gql.js",
+                        "../venia-ui/lib/**/storeSwitcher.gql.js"
```

Comment:
What is the reason for ignoring the store switcher queries?

Reply:
It contains query only available since Magento 2.4.1

Reply:
Hmm is there a way we can indicate a comment in this page or do we just have to remember to remove these when 2.4.1 is released?

Reply:
Add comment to `storeSwitcher.gql.js`